### PR TITLE
docs: update keywords and descriptions

### PR DESCRIPTION
**Pull request by yzeng25**
1. update keywords and descriptions
    1. unified beginning keywords `keywords: API7,Apache APISIX,API Gateway`
    2. added docs' categories (user case or blog) in keywords: 
        1. `keywords: API7,Apache APISIX,API Gateway,User Case,some_keywords`(for those in user case )
        2. `keywords: API7,Apache APISIX,API Gateway,blog,some_keywords` (for those in blog)
        3. `keywords: API7,Apache APISIX,API Gateway,User Case,blog,some_keywords`(for those in user case **and** blog)
    3. modified existing keywords
    4. added keywords and descriptions for all docs in usercase and blog, i noticed some older docs are missing keywords and descriptions, so I added keywords and descriptions to them.
2. minor fixes on wrong author name

### CODE REVIEW

File: _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: "A first look at Kubernetes Service APIs"
 avatar: "https://avatars.githubusercontent.com/u/4413028?s=460&u=e140a6d2bf19c426da6498b8888edc96509be649&v=4"
 author: "Jin Wei"
 href: "https://github.com/gxthrj"
+keywords: API7,Apache APISIX,API Gateway,blog,Kubernetes Service APIs
```
Comment (guoqqqi) [inline _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md:6]: Can blog also be used as a keyword?

File: _posts/blog/zh-CN/etcd3-support-HTTP-access-perfectly.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ author: spacewander
 href: "https://github.com/spacewander"
 avatar: "https://avatars.githubusercontent.com/u/4161644?v=4"
 date: 2021-06-30
-keywords: APISIX,etcd,API 网关,代理,Latency
+keywords: API7,Apache APISIX,API 网关,技术博客,代理,Latency
```
Comment (guoqqqi) [inline _posts/blog/zh-CN/etcd3-support-HTTP-access-perfectly.md:7]: Would it be better to add etcd here？

File: _posts/blog/zh-CN/jd-cloud.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: "支流科技与京东云携手，共筑 PaaS 新生态"
 author: 支流科技
 date: 2021-07-14
 keywords: APISIX,支流科技,京东云
+keywords: API7,Apache APISIX,API 网关,技术博客,京东云,支流科技
```
Comment (guoqqqi) [inline _posts/blog/zh-CN/jd-cloud.md:6]: Two keywords?

File: _posts/blog/zh-CN/kong-to-apisix.md [hunk 4]
```diff
@@ -5,6 +5,7 @@ avatar: "https://avatars.githubusercontent.com/u/34589752?v=4"
 href: "https://github.com/Yiyiyimu"
 date: 2021-08-05
 keywords: API 网关,APISIX,Kong,迁移,工具
+keywords: API7,Apache APISIX,API 网关,技术博客,Kong,迁移
```
Comment (guoqqqi) [inline _posts/blog/zh-CN/kong-to-apisix.md:8]: Two keywords?

File: _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md (hunk 1 above)
Comment (yzeng25) [reply _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md:6]: yes i think it is fine 

image

File: _posts/blog/zh-CN/etcd3-support-HTTP-access-perfectly.md (hunk 2 above)
Comment (yzeng25) [reply _posts/blog/zh-CN/etcd3-support-HTTP-access-perfectly.md:7]: yes, fixed

File: _posts/blog/zh-CN/jd-cloud.md (hunk 3 above)
Comment (yzeng25) [reply _posts/blog/zh-CN/jd-cloud.md:6]: deleted

File: _posts/blog/zh-CN/kong-to-apisix.md (hunk 4 above)
Comment (yzeng25) [reply _posts/blog/zh-CN/kong-to-apisix.md:8]: deleted

**Comment by guoqqqi** [comment]
cc @Baoyuantop have a look

**Review by Baoyuantop — APPROVED** [review]

**Pull request merged**